src/EspressoNitroTEEVerifier.sol

```diff
@@ -20,11 +21,14 @@ contract EspressoNitroTEEVerifier is NitroValidator, IEspressoNitroTEEVerifier,
     using CborDecode for bytes;
     using LibBytes for bytes;
     using LibCborElement for CborElement;
+    // PCR0 keccak hash for batch posters
 
-    // PCR0 keccak hash
-    mapping(bytes32 => bool) public registeredEnclaveHash;
-    // Registered signers
-    mapping(address => bool) public registeredSigners;
+    mapping(bytes32 => bool) public registeredBatchPosterEnclaveHashes;
+    mapping(bytes32 => bool) public registeredCaffNodeEnclaveHashes;
+    // Registered Caff Nodes
+    mapping(address => bool) public registeredCaffNodes;
+    // Registered Batch Posters
+    mapping(address => bool) public registeredBatchPosters;
     // Certificate Manager
     CertManager _certManager;
```

Comment:
would there be a need for a setter for this incase the certManager logic has to change and is thus deployed at another address?